css/deck.css

```diff
@@ -22,3 +22,9 @@ input[type=text]:focus+input[type=submit].icon-confirm,
 input[type=text]:hover+input[type=submit].icon-confirm {
 	border-color: var(--color-main-text) !important;
 }
+
+[class^="_tribute-container-autocomplete_"],
```

Comment:
Could this go to the CommentForm.vue file? We can add style block that is not scoped that then applies globally for this.